### PR TITLE
Removes the virologist job

### LINKED ISSUE #9774: Virology jobrole removal
Virology doesn't have enough content to exist as a standard job role, as most of the content in virology will result in a ban if it gets out.
Virus creation should be made into a part of medbay that is accessible to any decently ranking medical staff (doctors, chemists, geneticists, CMO) (paramedic doesn't need it as they work outside of medbay and should give people to heal over to medical staff except on skeleton crews).
The person who works the most on virology has expressed that they are fine with the job role being removed as it is better than the alternative of virology being changed dramatically.

### PR DESCRIPTION
**Pull request by Haliris**
## About The Pull Request

**This PR does not modify the virology systems nor the maps in any substance, all changes are focused on the Virologist job.**

Removes the virologist job role and all references of it in the game, including the action figurine.

Clothes were left as is as they are generated in closets on some maps, they remain for now as I consider this to touch on mapping, this PR only focuses on the job role.
I also didn't delete any sprites or other assets as of writing this, in case someone else wants to use them for their own projects.

## Why It's Good For The Game

The Virology systems don't provide enough content to support a standalone job such as virologist. The gameplay loop it provides pushes players to isolate themselves from medbay for most of the round, with most of the rewards not providing any value to ongoing events as they can't even be used most of the time.

The "good" viro as commonly referred to by some, is the viro player that spends an entire round toying with machines without involving other players, only to produce one virus that provides some gameplay advantage with nothing else to show. 

This isn't comparable to most science jobs as even those will either get their rewards faster, and will be able to capitalize on them more reliably without causing issues.

A "bad" viro player will produce a murderbone virus when they aren't supposed to, or will "accidentally" cause the release of a deadly virus, negatively impacting the round of everybody involved, in a manner that often isn't satisfying, even to other antag players or observers.

In both those archetypes, the systems are at fault in my opinion, as the virologist exists independently from the rest of the round. Unlike science, the virologist does not need mats, they only need power. 
In that sense, I find them a lot closer to miners, the difference being that mining systems are tied to the material/logistic systems, which are themselves built upon by many others in the game.

It must be pointed out that the removal of sentient disease likely pushed virologist further into a corner, as this was one of if not the only scenario where the crew would seek out the virologist to carry tasks and help with the grand scheme of things.
However, that antagonist was removed in part due to the virology systems not being interactive enough, alongside the antagonist systems being confusing/unfun to play with.
I believe that equation applies fully to the virologist role, hence this PR.

Additionally, removing this role opens up new avenues for design regarding the virology systems as they can now be approached without having to take into account a role that only has those going for it.

Going forward, the rest of medbay will be able to access virology and work on there if the players find it appropriate or simply want to.
I have set the medical doctors to have access to Virology at roundstart, as they are the jobs that make the most sense to me regarding viral studies and experimentation.

## Testing Photographs and Procedure

<details>

The game launches with no issues.
<summary>Screenshots&Videos</summary>

image

image

image

image

image

</details>

## Changelog
:cl:
del: Removed the virologist job and all of its references in the code and its associated spawners on the maps.
config: Removed the virologist mention in the config.
/:cl:

### CODE REVIEW

File: code/modules/crew_objectives/medical_objectives.dm
Comment (PowerfulBacon) [inline code/modules/crew_objectives/medical_objectives.dm]: this needs removal too otherwise it will still exist

Comment (Haliris) [reply code/modules/crew_objectives/medical_objectives.dm]: Cleaned it up.

File: code/game/objects/items/bedsheets.dm
Comment (Penwin0) [inline code/game/objects/items/bedsheets.dm]: This sentence doesn't make sense

Comment (Haliris) [reply code/game/objects/items/bedsheets.dm]: Fixed

**Comment by JixS4v** [comment]
10/10 PR, one step closer to virus creation removal

**Comment by Fronsis** [comment]
I mean, having most medical staff access to viro will lead to possible more accidental/griff plagues going out, alongside doctors going to viro to get their powergamer virus same way some do whenever there's no geneticists in lowpop and they get skeleton access

Sure, we can mark it as an administrative/player issue on that part however that leaves the rest then, which it's a gameplay loop issue that needs a similar shakedown compared to what lavaland removal/mining rework tries to do (get the miners on station, less ''fort'' gameplay, etc)

Believe it or not, some people just likes the idea of the ''virologist'' job, they experiment, create plagues, aid on curing them, etc

With this change this suddenly makes the doctor lean more into ''jack of all trades'' type since, if you really want to push it, they're capable of: surgeries, dealing with viruses, creating cures, producing vaccines, using genetics and chemistry(on lowpop), all types of medical treatments, etc.

It's quite different to the ''jack of all trades'' that Scientist are, which it's a whole 'nother can of worms

I don't see why a removal of the job is necessary because that will affect the few players that main virology that like the whole concept of it, now the regular MD will have a sudden random knowledge of virology which is just adding more to the ''jack of all trades'' type of players that Bee is against.

In the end, this won't solve much, what virology really needs it's a massive rework from the disease, the cures, the vaccines, etc

You'll just get MD players to go into Viro to get their funny virus akin to Scientist player that does toxins and then sits on xenobio to get their colorful slime extracts, however this will lead to more diseases from the hands of negligence, people learning and the griff individuals and leaving the later aside the CMO players rarely follow proper SoP to supervise what the viro does, now they need to check on every doctor that can enter and customize their own virus for themselves.

Some stuff will inherently generate more IC conflict, which is fine but from what i see this will outweight by the negative possibilities and doesn't really adress the main issue of the viro gameplay, if anything yes, you achieve that the player will leave the lab a little more, but most medbay players that are into virus-gaming can already get their mega powerful virus in less than 20 minutes so it'll be the equivalent of doing gene, get the fun mutations and never touch the lab again, it will also encourage players to want to handle everything by themselves compared to medbay cooperating to try to surpass a crisis or an scenario.

Feels like Paramedic removal discussion all over again just with worse possibilities waiting to happen.

Also what will happen to the hours? We all know players are proud to show the amount of hours they have on one job, does that remain as a ''legacy hour'' or it just get deleted?

**Comment by Haliris** [comment]
This feels like the paramedic removal discussion to you, because you are going on tangents about out of scope considerations while contradicting yourself.

Of course massively working on virology could be a way to improve the issues outlined, however no people willing to actually contribute instead of talking are considering this at the moment.
Secondly, this PR doesn't prevent such a rework. No assets are lost, and the core of the code that are the virology systems are left intact. Only the job dm file and its references are removed, which isn't a big deal in terms of work.

Lastly, either this PR does nothing, or it does, but you can't go back and forth on your comment, opening by explaining that this accomplishes nothing, while also explaining that it's bad because it does X or Y.

I don't see any directly relevant points raised.
Yes the alternative of putting way more work into a project will always yield a better result, that's just logical, so either put up a bounty or accept the fact that a removal is better.

**Comment by St0rmC4st3r** [comment]
I agree with this, except be so kind as to leave the figurine in.

**Comment by PowerfulBacon** [comment]
Decisions to remove roles and mechanics also have to take into account political/human factors like who has worked on the code and are they still around. We don't want to remove an entire feature if the coder that made it is still around so we have to take some compromises which in this case is removing the job role but keeping most of the mechanics.

There may be some doctors which occassionally spend an entire round in virology, but at least more people have access to the lab which makes them a lot less isolated and they still have the responsibilities of a doctor so virus manufactoring will mainly be for antagonists or as a side-job.

**Comment by EvilDragonfiend** [comment]
isn't it better for the job to have more playtime requirement than removing it?

**Comment by PowerfulBacon** [comment]
No. The problems with virology are due to the jobs gameplay and duties, not due to playtime requirements or griefing

**Comment by JikkaJoestar** [comment]
The proposed changes remind me of how the scientist job works, with multiple possible tasks that a scientist can dedicate themselves to. The only potential issue is doctors staying in virology and, especially on lowpop, not being available or willing to treat patients. I do agree with the removal of the virology role in general, though.

**Comment by PowerfulBacon** [comment]
In an admin meeting I think we all agreed that it was better to fully remove the virology room as well and not make it part of something that the doctor can do.

**Comment by JikkaJoestar** [comment]
> In an admin meeting I think we all agreed that it was better to fully remove the virology room as well and not make it part of something that the doctor can do.

In that case we might need to delay this PR and make a bounty for the mapping removals. Some items, such as the virus extrapolator, would be necessary somewhere else if we aren't going to remove viruses as a whole.

**Comment by JixS4v** [comment]
> In an admin meeting I think we all agreed that it was better to fully remove the virology room as well and not make it part of something that the doctor can do.

Yes, just remove the virology mechanic roundstart, maybe make the stuff orderable through cargo or something

**Comment by zeskorion** [comment]
> Decisions to remove roles and mechanics also have to take into account political/human factors like who has worked on the code and are they still around. We don't want to remove an entire feature if the coder that made it is still around so we have to take some compromises which in this case is removing the job role but keeping most of the mecichanics.
> 
> There may be some doctors which occassionally spend an entire round in virology, but at least more people have access to the lab which makes them a lot less isolated and they still have the responsibilities of a doctor so virus manufactoring will mainly be for antagonists or as a side-job.


I'll start working, soon, on a complete removal of player-facing virology. Virus engineering will not be made impossible, but it will be made difficult, and I will re-tune viruses and associated events to make them more of a rare hazard, than a powergame mechanic

**Review by PestoVerde322 — COMMENTED** [review]
Mapwise it's fine since it removes the spawners, but unclear why also remove the Virologist figurine since that's a collectible.

File: code/modules/reagents/reagent_containers/bottle.dm
Comment (Rukofamicom) [inline code/modules/reagents/reagent_containers/bottle.dm]: "Robust member of medical staff" Sounds better. I'm sure there's other ways to word it well too, but this just sounds off.

**Review by Rukofamicom — APPROVED** [review]
Removal looks clean from what I can see, though I do think the virologist figures and AI Vox could stay because they aren't causing any harm.

File: code/modules/reagents/reagent_containers/bottle.dm
Comment (Tsar-Salat) [reply code/modules/reagents/reagent_containers/bottle.dm]: Just say doctor. Everyone will get the meaning

File: tools/UpdatePaths/Scripts/7550_pdas_to_tabs.txt
Comment (itsmeow) [inline tools/UpdatePaths/Scripts/7550_pdas_to_tabs.txt]: Don't remove this, it's an important migration for legacy reasons

File: code/modules/mob/living/simple_animal/friendly/hamster.dm
Comment (itsmeow) [inline code/modules/mob/living/simple_animal/friendly/hamster.dm]: soul removed

File: code/modules/mob/living/silicon/ai/vox_sounds.dm
Comment (itsmeow) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: actually remove this file or put it back

File: code/modules/crew_objectives/medical_objectives.dm
Comment (itsmeow) [inline code/modules/crew_objectives/medical_objectives.dm]: you could give this to doctors

File: code/__DEFINES/jobs.dm
Comment (itsmeow) [inline code/__DEFINES/jobs.dm]: remove the viro hud sprite

**Review by itsmeow — CHANGES_REQUESTED** [review]
We need to do some work on the job playtime system so that a theoretical player with viro medical hours can still play CMO - and playtime doesn't inexplicably drop. I will open a PR for this I think.

**Comment by EvilDragonfiend** [comment]
also, do not merge without admin to change wiki pages.

File: code/modules/mob/living/simple_animal/friendly/hamster.dm
Comment (Haliris) [reply code/modules/mob/living/simple_animal/friendly/hamster.dm]: There wouldn't be any virologists to lynch, also it's a random message that we miss all the time

File: code/modules/mob/living/silicon/ai/vox_sounds.dm
Comment (Haliris) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Reverted this removal

File: code/modules/crew_objectives/medical_objectives.dm
Comment (Haliris) [reply code/modules/crew_objectives/medical_objectives.dm]: I assigned it to doctor instead

File: code/__DEFINES/jobs.dm
Comment (Haliris) [reply code/__DEFINES/jobs.dm]: Removed

File: tools/UpdatePaths/Scripts/7550_pdas_to_tabs.txt
Comment (Haliris) [reply tools/UpdatePaths/Scripts/7550_pdas_to_tabs.txt]: Reverted

File: code/modules/reagents/reagent_containers/bottle.dm
Comment (Haliris) [reply code/modules/reagents/reagent_containers/bottle.dm]: Changed

**Comment by Haliris** [comment]
I can revert the removal of the virologist figurine if it's really desired, I initially removed it because I considered it to be an empty reference without an associate jobrole.

**Comment by mystery3525** [comment]
> also, do not merge without an admin to change wiki pages.

If it is just removing the virologist from SoP then [they want to be notified after the merge](https://github.com/BeeStation/BeeStation-Hornet/pull/9825#issuecomment-1728428489).

**Comment by Haliris** [comment]
It's not dead by the way, I just did a git desktop moment trying to solve the conflict and need to figure out this shit

File: tools/UpdatePaths/Scripts/7550_pdas_to_tabs.txt
Comment (PowerfulBacon) [reply tools/UpdatePaths/Scripts/7550_pdas_to_tabs.txt]: It would need to point to a typepath that actually exists though

**Pull request closed without merging**
